src/main/java/uk/gov/hmcts/reform/cpo/controllers/CasePaymentOrdersController.java

```diff
@@ -51,25 +52,68 @@ public CasePaymentOrdersController(CasePaymentOrdersService casePaymentOrdersSer
-                                                             @RequestParam("cases-ids") Optional<List<String>> casesId,
-                                                             @RequestParam("pageSize") Optional<Integer> pageSize,
-                                                             @RequestParam("pageNumber") Optional<Integer> pageNumber
+
+    @GetMapping(value = "case-payment-orders", produces = {"application/json"})
+    @ApiOperation(value = "Get payment orders for a case", notes = "Get payment orders for a case")
+    @ApiResponses({
+        @ApiResponse(
+            code = 200,
+            message = ""
+        ),
+        @ApiResponse(
+            code = 400,
+            message = "One or more of the following reasons:"
+                + "\n1) " + ValidationError.CPO_FILER_ERROR
+                + "\n2) " + ValidationError.CASE_ID_INVALID
+                + "\n3) " + ValidationError.ID_INVALID,
+            response = String.class,
+            examples = @Example({
+                @ExampleProperty(
+                    value = "{\n"
+                        + "   \"status\": \"400\",\n"
+                        + "   \"error\": \"Bad Request\",\n"
+                        + "   \"message\": \"" + ValidationError.ARGUMENT_NOT_VALID + "\",\n"
+                        + "   \"path\": \"" + CASE_PAYMENT_ORDERS_PATH + "\",\n"
+                        + "   \"details\": [ \""  + ValidationError.CASE_ID_INVALID + "\" ]\n"
+                        + "}",
+                    mediaType = APPLICATION_JSON_VALUE)
+            })
+        ),
+        @ApiResponse(
+            code = 401,
+            message = AuthError.AUTHENTICATION_TOKEN_INVALID
+        ),
+        @ApiResponse(
+            code = 403,
+            message = AuthError.UNAUTHORISED_S2S_SERVICE
+        )
+    })
+    public Page<CasePaymentOrder> getCasePaymentOrders(@ApiParam("list of case payment orders ids")
+                                                       @ValidCpoId
+                                                       @RequestParam(name = "ids", required = false)
+                                                       Optional<List<String>> ids,
+                                                       @ApiParam("list of ccd case reference numbers")
+                                                       @ValidCaseId
+                                                       @RequestParam(name = "case-ids", required = false)
+                                                       Optional<List<String>> caseIds,
+                                                       @RequestParam(name = "pageSize", required = false)
+                                                       Optional<Integer> pageSize,
+                                                       @RequestParam(name = "pageNumber", required = false)
+                                                       Optional<Integer> pageNumber
```

Comment:
I've seen that it's also possible to have a `Pageable` object in the controller. Have we considered that?
https://reflectoring.io/spring-boot-paging/#accepting-a-pageable-parameter